horizons/network/server.py

```diff
@@ -298,61 +300,73 @@ def onreceive(self, event):
 			self.error(player, e.message)
 			return
 		except network.PacketTooLarge as e:
-			logging.warning("[RECEIVE] Per packet size exceeded from %s: %s" % (player, e))
+			logging.warning("[RECEIVE] Per packet size exceeded from "
+				"{0!s}: {1!s}".format(player, e))
 			self.fatalerror(player, __("You've exceeded the per packet size.") + " " +
 			                        __("This should never happen. "
 			                           "Please contact us or file a bug report.") +
 			                        " " + str(e))
 			return
 		except Exception as e:
-			logging.warning("[RECEIVE] Unknown or malformed packet from %s: %s!" % (player, e))
+			logging.warning("[RECEIVE] Unknown or malformed packet "
+				"from {0!s}: {1!s}".format(player, e))
 			self.fatalerror(player, __("Unknown or malformed packet. Please check your game version"))
 			return
 
 		# session id check
 		if packet.sid != player.sid:
-			logging.warning("[RECEIVE] Invalid session id for player %s (%s vs %s)!" % (peer.address, packet.sid, player.sid))
-			self.fatalerror(player, __("Invalid/Unknown session")) # this will trigger ondisconnect() for cleanup
+			logging.warning(
+				"[RECEIVE] Invalid session id for player {0!s} "
+				"({1!s} vs {2!s})!"
+				.format(peer.address, packet.sid, player.sid))
+			self.fatalerror(player, __("Invalid/Unknown session"))
+			# this will trigger ondisconnect() for cleanup
```

Comment:
If you move the comment, it should definitely go before the line that is meant, not after.